src/features/utils/ansibleCfg.ts

```diff
@@ -0,0 +1,90 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import untildify from 'untildify';
+import * as ini from 'ini';
+
+// Get rootPath based on multi-workspace API
+export function getRootPath(editorDocumentUri: vscode.Uri): string | undefined {
+  let rootPath: string | undefined = undefined;
+
+  if (!!vscode.workspace.workspaceFolders) {
+    rootPath = vscode.workspace.workspaceFolders.length
+      ? vscode.workspace.workspaceFolders[0].name
+      : undefined;
+  }
+
+  if (!!vscode.workspace.getWorkspaceFolder) {
```

Comment:
A semantically correct check would be
```suggestion
  if (typeof vscode.workspace.getWorkspaceFolder === 'function') {
```